_licenses/postgresql.txt

```diff
@@ -0,0 +1,50 @@
+---
+title: PostgreSQL License
+spdx-id: PostgreSQL
+source: https://opensource.org/licenses/PostgreSQL
+featured: false
+hidden: true
+
+description: A very short, BSD-style license, used specifically for PostgreSQL.  
+
+how: To use it, say that it is The PostgreSQL License, and then substitute the copyright year and name of the copyright holder into the body of the license. Then put the license into a prominent file ("COPYRIGHT", "LICENSE" or "COPYING" are common names for this file) in your software distribution.
+
+using:
+  - PostgreSQL: https://www.postgresql.org/about/licence/
+  - PGAudit: https://github.com/pgaudit/pgaudit/blob/master/LICENSE
+  - .NET Access to PostgreSQL: https://github.com/npgsql/npgsql/blob/dev/LICENSE.txt
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - private-use
+
+conditions:
+  - include-copyright
+
+limitations:
+  - liability
+  - warranty
+
+---
+
+The PostgreSQL Licence (PostgreSQL)
+
+Copyright (c) [year] [fullname]
+
+Permission to use, copy, modify, and distribute this software and its
+documentation for any purpose, without fee, and without a written agreement is
+hereby granted, provided that the above copyright notice and this paragraph
+and the following two paragraphs appear in all copies.
+
+IN NO EVENT SHALL [fullname] BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT,
```

Comment:
Is this (and fullname in next 2 paragraphs) typically replaced in the wild? Old versions of similar licenses were typically templated throughout, but generic terms (like "copyright holder") are now typically found in the license body.